### PR TITLE
requestmapping url change and getmapping endpoint change

### PR DESCRIPTION
**Pull request by pratikmhaske007**
Description : 
requestmapping url change and getmapping endpoint change.

Story/Defect :
(https://github.com/vivekdhole03/SpringBootApplicationDataJPA/pull/6)

### CODE REVIEW

File: src/main/java/com/csi/controller/ProductController.java [hunk 1]
```diff
@@ -13,13 +13,13 @@
 import java.util.List;
 
 @RestController
-@RequestMapping("/V1")
+@RequestMapping("/V23")
```
Comment (ganudhole) [inline src/main/java/com/csi/controller/ProductController.java]: please change v53

File: src/main/java/com/csi/controller/ProductController.java [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
 public class ProductController {
 
     @Autowired
     ProductService productServiceImpl;
 
-    @GetMapping("/getalldata")
+    @GetMapping("/viewalldata")
```
Comment (vivekdhole03) [inline src/main/java/com/csi/controller/ProductController.java]: Can we change viewalldata to viewAllData ?

File: src/main/java/com/csi/controller/ProductController.java (hunk 1 above)
Comment (pratikmhaske007) [reply src/main/java/com/csi/controller/ProductController.java]: changes done

File: src/main/java/com/csi/controller/ProductController.java (hunk 2 above)
Comment (pratikmhaske007) [reply src/main/java/com/csi/controller/ProductController.java]: changes done

**Review by ganudhole — APPROVED** [review]

**Pull request merged**